corehq/apps/reports/standard/users/reports.py

```diff
@@ -54,19 +55,21 @@ def get_primary_properties(cls, domain):
         needing to click "See More".
         """
         if domain_has_privilege(domain, privileges.APP_USER_PROFILES):
-            user_data_label = _("Profile or User Data")
+            user_data_label = _("profile or user data")
         else:
-            user_data_label = _("User Data")
+            user_data_label = _("user data")
         return {
-            "username": _("Username"),
-            "email": _("Email"),
-            "domain": _("Project"),
-            "is_active": _("Is Active"),
-            "language": _("Language"),
-            "phone_numbers": _("Phone Numbers"),
-            "location_id": _("Primary Location (mobile users only)"),
+            "username": _("username"),
+            "role_id": _("role"),
+            "email": _("email"),
+            "domain": _("project"),
+            "is_active": _("is active"),
+            "language": _("language"),
+            "phone_numbers": _("phone numbers"),
+            "location_id": _("location"),
+            "commcare_location_id": _("location"),
```

Comment:
its still not clear to me why this is needed?
we are already filtering for `location_id` which is same as `commcare_location_id` and plus `commcare_location_id` is set in user_data, so its not available as a top level field to filter on anyway.

Reply:
I agree, looking at it now I don't think it's needed. Initially I didn't realize they were the same, and was converting both to location name. Since, like you said, `commcare_location_id` isn't available for that top level filtering and will render the same location name anyway, it seems less redundant to have `commcare_location_id` display the id string as opposed to location name.